trace-test-application/src/androidTest/java/io/bitrise/trace/testapp/ActivityUiTest.java

```diff
@@ -25,7 +27,7 @@
     @Test
     public void activityStateTest() {
         final IndexActivityScreen indexActivityScreen = new IndexActivityScreen(uiDevice);
-
+        assertNotNull(indexActivityScreen.getButtonUiTestsLabel());
```

Comment:
I was wondering if we should add assertions for the app launch (launchTraceTestApp method) as this test is for the state of the activities of the app, not about the UI.

Reply:
Interesting - can you explain a bit more what you mean by that?

Reply:
So method launchTraceTestApp returns a Boolean value if the launch was successful or not, we could add assertion for that method.

Reply:
Ah. Sure. I've updated ActivityUiTest to use this - however the other tests (ApplicationStartUpUiTest and FragmentUiTest) will still use the checking the button exists. Does that sound okay?

Reply:
Sure.